model/user.go

```diff
@@ -85,6 +85,7 @@ type User struct {
 	LastActivityAt         int64     `db:"-" json:"last_activity_at,omitempty"`
 	IsBot                  bool      `db:"-" json:"is_bot,omitempty"`
 	BotDescription         string    `db:"-" json:"bot_description,omitempty"`
+	BotLastIconUpdate      int64     `db:"-" json:"bot_last_icon_update,omitempty"`
```

Comment:
@ali-farooq0 Do you know if this is needed here or if the field in `model.User` is enough?

Reply:
It is needed so the webapp can get access to it without doing any extra work. I know it's also added to `model.Bot` but currently the webapp doesn't fetch bot details. This is the same reason we have `BotDescription` here as well.

Reply:
So this is just for performance? I'm not convinced :thinking:

Reply:
@hanzei no, not for performance reasons, mainly for consistency reasons. Currently some of the commonly used `bot` fields are already in this `User` model. In the webapp we're currently not fetching or storing bot details. To switch to fetching those bot details will not be a trivial task, as it requires some thought and effort on how it'll be done. Do we create a new model in the server and do it there or pull all the fields in the webapp and do it there, etc.

The reason I proposed adding  `BotLastIconUpdate` is because we already have `IsBot` and `BotDescription` here, adding another field won't make it any worse. Personally I would want to remove `bot` specific fields from this model and create another model that contains all the necessary details for user + bot or anything else that can be sent down to the webapp for it to consume, but that's a non-trivial change unfortunately. So going with this compromised approach for now.

Reply:
I see. Thanks for the explanation.